index.html

```diff
@@ -10,5 +10,13 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech-div">
+    <span class="tech-span">
+      <img class="img-tech" src="assets/react_icon.svg" alt="">
+      <h2 class="tech-descr">ReacjJS</h2>
+      <h2 class="tech-descr"> <strong>Release: 2013</strong></h2>
+      <h2 class="tech-descr"> React is a declarative,<br> component-based <br> JavaScript library for <br> building user interfaces </h2>
```

Comment:
Remove all those `<br>`. It doesnt matter if text break is not exactly the same the picture, but you have many tools to make text wrap and center